chat-plugins/trivia.js

```diff
@@ -1675,11 +1676,7 @@ module.exports = {
 	commands: {
 		trivia: commands,
 		ta: commands.answer,
-		wl: {
-			add: commands.add,
-			submit: commands.submit,
-			qs: commands.qs,
-		},
+		wl: commands,
```

Comment:
`/wla` as an alias for `/wl answer` instead of having to use `/ta`would make more sense

Reply:
... ``/wl`` is an alias for ``/weeklock`` :|